app/components/Button/index.jsx

```diff
@@ -0,0 +1,38 @@
+import cx from 'classnames'
+import React from 'react'
```

Comment:
I will make this change, but leave it on another branch so I don't wipe out any review comments.